addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1582,13 +1582,31 @@ options.registry.topMenuColor = options.registry.colorpicker.extend({
 /**
  * Handles the edition of snippet's anchor name.
  */
-options.registry.anchorName = options.Class.extend({
-    xmlDependencies: ['/website/static/src/xml/website.editor.xml'],
+options.registry.anchor = options.Class.extend({
 
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    start: function () {
+        // Initialize tooltip, use a DP to hide it 1500ms after the last click
+        this.$button = this.$el.find('we-button');
+
+        // Generate anchor and copy it to clipboard on click, show the tooltip on success
+        const clipboard = new ClipboardJS(this.$button[0], {text: () => this._getAnchorLink()});
+        clipboard.on('success', () => {
+            this.displayNotification({
+              title: _t("Copied !"),
+              message: _t("The anchor has been copied to your clipboard.<br>Link: ") + decodeURIComponent(this._getAnchorLink()),
```

Comment:
Avoid concatenation of translated string in any case (even if it seems pretty safe here, you don't know all languages and if a careless developer changes the original string it might get less safe).

-> Decode the URL before the `displayNotification` call and put it in a var and print that var in the translated string.